pkg/inventory/main_test.go

```diff
@@ -0,0 +1,19 @@
+// Copyright 2022 The Kubernetes Authors.
```

Comment:
This test file could be pulled out. It's not a big deal if there's only one, but it's completely independent. It just allows the package to be run with logging verbosity.